backend/api/tests/test_project.py

```diff
@@ -866,7 +866,7 @@ def test_submission_status_groups_submitted_and_passed_checks(self):
 
         self.assertEqual(
             content_json["status"],
-            {"non_empty_groups": 3, "groups_submitted": 2, "submissions_passed": 2},
+            {"non_empty_groups": 3, "groups_submitted": 2, "extra_checks_passed": 0, "structure_checks_passed": 0},
```

Comment:
Why is this structure_check_passed here 0 if the name of this test includes "and_passed_tests"?

Reply:
Because it now checks that the tests shouldn't pass. In the previous test the tests still passed by default. That is not the case anymore.